app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -82,7 +82,7 @@ export class MarksPanel extends React.Component {
   };
 
   renderMarkComponent = (markData) => {
-    const key = `${markData.criterion_type}-${markData.id}`;
```

Comment:
These lines should remain unchanged.

https://github.com/MarkUsProject/Markus/blob/a980cda3864ad7047ec7a4d3b0d85497077cdf77/app/assets/javascripts/Components/Result/marks_panel.jsx#L102-L109